cmd/gdsutil/main.go

```diff
@@ -1034,13 +1034,13 @@ func registerRepair(c *cli.Context) (err error) {
 		}
 
 		// Connect to secret manager
-		var secrets *gds.SecretManager
-		if secrets, err = gds.NewSecretManager(conf.Secrets); err != nil {
+		var sm *secrets.SecretManager
+		if sm, err = secrets.New(conf.Secrets); err != nil {
 			return cli.NewExitError(err, 1)
 		}
```

Comment:
The `SecretManager` has been moved to its own package so that it could be mocked when Google application credentials were not available. The mock code and interface was copied from Whisper - but none of the `SecretManager` code was changed! The power of interfaces! :-) 

There is an opportunity to write better tests for this now, FYI - happy to discuss it and add a story for it.